recipes/minikube_setup/configure.py

```
@@ -0,0 +1,148 @@
+    cmd = ["minikube", "start", "--driver", "docker", "--container-runtime", "docker", "--gpus", "all"]
+    check_call(cmd)
+    logger.info("Minikube started")
+
+    current_context = check_output(["kubectl", "config", "current-context"], text=True).strip()
+    logger.info(f"Current context: {current_context}")
+
+    if current_context != "minikube":
+        logger.info("Switching context to minikube")
+        check_call(["kubectl", "config", "use-context", "minikube"])
+
+    current_context = check_output(["kubectl", "config", "current-context"], text=True).strip()
```

Ruff (0.15.21)

[error] Starting a process with a partial executable path

(S607)